src/qt/forms/mrcrequestpage.ui

```diff
@@ -226,19 +226,26 @@
            <string>MRC Fee Boost</string>
           </property>
           <property name="alignment">
-           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
+           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mrcFeeBoostRaiseToMinimumButton">
+          <property name="toolTip">
+           <string>This will automatically boost the mrc fee you are paying to get your MRC request in the queue.</string>
+          </property>
```

Comment:
shouldn't "MRC" be capitalized in both instances?

Reply:
I suppose we should be consistent! :)